.github/CODEOWNERS

```diff
@@ -0,0 +1,64 @@
+# This list auto requests reviews from the specified org members
+# when a PR that modifies the file in question is opened
+# This list is alphabetized by User -> Filename KEEP IT THAT WAY
+
+# ChangelingRain
+
+/code/__DEFINES/clockcult.dm @ChangelingRain
```

Comment:
is joan active?

Reply:
vOv just copied the same stuff she asked for last time

Reply:
@ChangelingRain still interested?

Reply:
12:36 <%oranges> Joan: are you still wanting code owner pings?
12:36 <%Joan> yes